docs/releases/1.11.txt

```diff
@@ -673,3 +673,7 @@ Miscellaneous
   ``ETag`` header to responses regardless of the setting. ``CommonMiddleware``
   and ``django.utils.cache.patch_response_headers()`` will no longer set ETags
   when the deprecation ends.
+
+* :class:`~django.middleware.http.ConditionalGetMiddleware` no longer sets the
```

Comment:
Modify the existing note: "* ``ConditionalGetMiddleware`` no longer sets the ``Date`` header as Web
  servers set that header." This section is actually "deprecated features".